types/minio/index.d.ts

```diff
@@ -80,14 +79,12 @@ export class Client {
     constructor(options: ClientOptions);
 
     // Bucket operations
-    makeBucket(bucketName: string, callback: NoResultCallback): void;
     makeBucket(bucketName: string, region: Region, callback: NoResultCallback): void;
-    makeBucket(bucketName: string, region?: Region): Promise<void>;
```

Comment:
we are going to make region optional https://github.com/minio/minio-js/pull/672 hence this PR is not needed

Reply:
We will also change the lastModified to `Date` @barinbritva

Reply:
This typings are for version 4. In version 4 region is required and lastModified is string.
I will write all needed changes for next minio-js version when it released. I think it will be right.

Reply:
@barinbritva, we have decided that lastModified can be kept as `string` here until the relevant changes have been made.